app/models/case/sar/offender.rb

```diff
@@ -88,6 +92,12 @@ def searchable_fields_and_ranks
     third_party_recipient: 'third_party_recipient',
   }
 
+  enum further_actions_required: {
```

Comment:
Can these be translated (if it's not a pain!)

Reply:
This one don't need, the UI component will do the job,  if somehow the wording is different from the current one in the future which I really doubt, we can put them into translate later without touch this part.